FAQ: explain setting e-mail subject and signature (#793)
deltachat/deltachat-pages: en/help.md

  device.

### Can I set the E-Mail Subject with Delta Chat?

For most messages, Delta Chat sets the subject to
"New Message from Alice" automatically,
or "..." if it is encrypted.
The exception are group chats,
where the subject is the Chat Title.

So if you want to set the subject yourself,
for example for a formal e-mail to a business contact,
you can create a group
with only yourself and the recipient(s),
set the Chat Title to the Subject you want,
and then send your e-mail as the first message.
If the recipients don't use Delta Chat,
it will look like a normal,
formal e-mail to them.
You will get bonus professionalism points
if you set a [signature](#signature).

> ### Can I set an E-Mail Signature with Delta Chat? {#signature}

Reviewer: ```suggestion
### Can I set a Signature (Status, Motto) with Delta Chat? {#signature}
```

i would leave out "E-Mail" in the question, it is explained below.

not sure about the brackets, though. but maybe it makes things clearer in case ppl do not know the term "Signature". for nerds, it also makes clear, that this is not a crypto thing :) (which is called "signature" as well at other places of the help)
